tests/unit_tests/template.py

```diff
@@ -0,0 +1,834 @@
+import json
+import re
+import subprocess
+import os
+import sqlite3 as lite
+import copy
+
+from nose.tools import assert_in, assert_true, assert_equals
+
+direc = os.listdir('./')
+for item in direc:
+    if item.endswith('.sqlite') or item.endswith('.json'):
+        os.remove(os.path.join('./', item))
+
+env = dict(os.environ)
+env['PYTHONPATH'] = "."
```

Comment:
PEP8: Global variables should be ALL_UPPERCASE

Reply:
Also, I am not sure it is safe to rewrite the entire PYTHONPATH.  You should instead preprend.

```python
ENV['PYTHONPATH'] = '.:' + ENV.get('PYTHONPATH', '')
```